### PR TITLE
[RLlib] Fix MultiCallbacks class: To be used only with utility function that returns a class to use in the config.

### PR DESCRIPTION
**Pull request by sven1977**
Fix MultiCallbacks class: To be used only with utility function that returns a class to use in the config.
* Thus far, when using MultiCallbacks, the user would have to create a callable object (instance) of MultiCallbacks, passing the list of sub-classes to the constructor, then having to call the object's `__call__` method to instantiate these sub-classes.
* This is counterintuitive as Callbacks should always be passed as types (classes) to the config's `.callbacks([your custom class here])` method.
* This PR introcudes a helper utility function that creates the proper multi callbacks class for you. Synopsis is:

```
from ray.rllib.algorithms.ppo import PPOConfig
from ray.rllib.algorithms.callbacks import make_multi_callbacks

config = (
    PPOConfig()
    .environment("CartPole-v1")
    .callbacks(make_multi_callbacks([MyCallbackCls1, MyCallbackCls2])
)

config.build()
```

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by kouroshHakha — APPROVED** [review]
Awesome 👍

**Comment by richardliaw** [comment]
FYI, @zhe-thoughts needs to approve all changes this week due to branch cut.

**Comment by zhe-thoughts** [comment]
OK here's my retrospective approval

Please tag me and wait for an approval before merging (until we actually cut the 2.4.0 branch). Thanks!

**Pull request merged**
